src/components/search-form/__tests__/SearchForm.test.js

```diff
@@ -67,6 +67,23 @@ describe('components/search-form/SearchForm', () => {
         sinon.assert.calledWithMatch(onSubmitSpy, 'cheese', event);
     });
 
+    test('should call onsubmit on search icon submit event', () => {
+        const onSubmitSpy = sinon.spy();
+        const event = {
+            target: {
+                elements: [
+                    {
+                        value: 'cheese',
+                    },
+                ],
+            },
+        };
+        const wrapper = mount(<SearchForm onSubmit={onSubmitSpy} placeholder="search" value="cheese" />);
```

Comment:
Shallow doesn't work, because the form itself is mounted as a component
```
<LoadableSearchActions
    isLoading={isLoading}
    loadingIndicatorProps={{
        className: 'search-form-loading-indicator',
    }}
/>
```